### PR TITLE
import resolver: Don't attempt to resolve non-matching modules

### PR DESCRIPTION
**Pull request by andrewwormald**

### CODE REVIEW

File: example/run.sh [hunk 1]
```diff
@@ -0,0 +1,2 @@
+# Command used to generate the example
```
Comment (echarrod) [inline example/run.sh:1]: Could we add this to the README.md? I guess it make more sense in a CONTRIBUTING.md file, once we have one

File: generator/generator.go [hunk 2]
```diff
@@ -304,7 +305,7 @@ func switchToTypescriptType(typ string) string {
 		if strings.HasPrefix(typ, "map") { // map[int64]bool
 			typ = strings.TrimPrefix(typ, "map[") // int64]bool
 
-			keyAndValueTypes := []string{} // 0 - key, 1 - value of a map
+			keyAndValueTypes := []string{}              // 0 - key, 1 - value of a map
```
Comment (echarrod) [inline generator/generator.go:308]: This formatting looks a little strange 🤔

Comment (andrewwormald) [reply generator/generator.go]: This is the formatter being run 🤷‍♂️

Comment (echarrod) [reply generator/generator.go]: Think it only gets run on staged files, I'll do a repo update separately 👍

**Review by echarrod — APPROVED** [review]

**Pull request merged**